resources/installers/install-munajjam-windows.ps1

```
@@ -0,0 +1,99 @@
+if (-not (Test-Path (Join-Path $RepoDir ".git"))) {
+  Write-Host "Cloning Munajjam repository..."
+  git clone --depth 1 --branch $RepoRef $RepoUrl $RepoDir
+} else {
+  Write-Host "Updating managed Munajjam repository..."
+  git -C $RepoDir fetch origin $RepoRef --depth 1
+  git -C $RepoDir checkout $RepoRef
+  git -C $RepoDir pull --ff-only origin $RepoRef
+}
+
+if (-not (Test-Path $VenvDir)) {
+  Write-Host "Creating Python virtual environment..."
```

PSScriptAnalyzer (1.24.0)

[warning] File 'install-munajjam-windows.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)